compiler-plugin/src/main/java/io/ballerina/stdlib/mi/plugin/AnnotationAnalysisTask.java

```diff
@@ -57,15 +61,36 @@ private static void setParameters(FunctionSymbol functionSymbol, Component compo
 
             for (int i = 0; i < noOfParams; i++) {
                 ParameterSymbol parameterSymbol = parameterSymbols.get(i);
-                if (parameterSymbol.getName().isEmpty()) continue;
-                Param param = new Param(Integer.toString(i), parameterSymbol.getName().get());
-                component.addBalFuncParams(param);
+                Optional<String> optParamName = parameterSymbol.getName();
+                if (optParamName.isEmpty()) continue;
```

Comment:
We don't follow this code style. Also, check check-style is configured.

Reply:
Fixed